src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-darPath.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Path for newly created dar file relative to workspace. Jenkin's environment variables can be used to specify this value.
```

Comment:
Path for the newly created DAR file, relative to the workspace. Jenkins environment variables can be used to specify this value.